test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -316,6 +316,162 @@ proc ::AlgorandGoal::WaitForAccountBalance { WALLET_NAME ACCOUNT_ADDRESS EXPECTE
     return $ACCOUNT_BALANCE
 }
 
+# Create an asset
+proc ::AlgorandGoal::AssetCreate { CREATOR WALLET_NAME WALLET_PASSWORD TOTAL_SUPPLY UNIT_NAME TEST_PRIMARY_NODE_DIR } {
+    set timeout 40
+    if { [ catch {
+        spawn goal asset create -d $TEST_PRIMARY_NODE_DIR -w $WALLET_NAME --creator $CREATOR --total $TOTAL_SUPPLY --unitname $UNIT_NAME
+        expect {
+            timeout { ::AlgorandGoal::Abort "Failed to create asset"  }
+	    "Please enter the password for wallet '$WALLET_NAME':" { send "$WALLET_PASSWORD\r" }
+        }
+    } EXCEPTION ] } {
+       ::AlgorandGoal::Abort "ERROR in AssetCreate: $EXCEPTION"
+    }
+}
+
+# Transfer asset
+proc ::AlgorandGoal::AssetTransfer { WALLET_NAME WALLET_PASSWORD FROM_ADDR TO_ADDR ASSET_ID ASSET_AMOUNT TEST_PRIMARY_NODE_DIR} {
+    if { [ catch {
+        spawn goal asset send -d $TEST_PRIMARY_NODE_DIR -w $WALLET_NAME --from $FROM_ADDR --to $TO_ADDR --assetid $ASSET_ID --amount $ASSET_AMOUNT 
+        expect {
+            timeout { ::AlgorandGoal::Abort "Failed to send asset"  }
```

Comment:
Could you change the error text to indicate that the error was triggered by a timeout ?